oh-my-bash.sh

```diff
@@ -31,6 +31,11 @@ for config_file in $OSH/lib/*.sh; do
   source $config_file
 done
 
+# Load colors first so they can be use in base theme
+source "${OSH}/themes/colours.theme.sh"
```

Comment:
@pokowaka Can you please explain why there is a change inside this bootstrap while you are adding new plugin?

Reply:
@pokowaka Themes should be loaded after plugins, completions, aliases...etc because only theme is something very specified but plugin is for global usage.

Reply:
@pokowaka any eta?

Reply:
Another solution is to move `safe_append_prompt_command` to `lib/` (e.g. `lib/base.sh` or `lib/utils.sh`). I don't think `safe_append_prompt_command` should be considered a part of the utilities for themes.

Reply:
I have moved the definition of `safe_append_prompt_command ` to `lib/utils.sh`, and then renamed it to `_omb_util_append_prompt_command` in commit a3b8e5f.